bindings/java/src/org/sleuthkit/datamodel/OsAccount.java

```diff
@@ -409,7 +362,7 @@ public OsAccountDbStatus getOsAccountDbStatus() {
 	 *
 	 * @throws TskCoreException
 	 */
-	public List<OsAccountInstance> getOsAccountInstances() throws TskCoreException {
+	public synchronized List<OsAccountInstance> getOsAccountInstances() throws TskCoreException {
```

Comment:
It looks like we don't have a way to set OS account instances, either via a setter or the constructor. Perhaps we need to add a synchronized setInstancesInternal() method?